bugbug/repository.py

```diff
@@ -1010,12 +1011,10 @@ def clone(repo_dir, url="https://hg.mozilla.org/mozilla-central"):
     clean(repo_dir)
 
 
-def apply_stack(repo_dir, stack, branch, default_base):
+def apply_stack(repo_dir, stack, branch, default_base="tip"):
```

Comment:
Since it's always "tip" now, we can just remove the parameter to simplify.

Reply:
Done